packages/doc-site/styleguide.config.js

```diff
@@ -6,119 +6,123 @@ function kebabize(string) {
   }
   // uppercase after a non-uppercase or uppercase before non-uppercase
   const upper = /(?<!\p{Uppercase_Letter})\p{Uppercase_Letter}|\p{Uppercase_Letter}(?!\p{Uppercase_Letter})/gu;
-  return string.replace(upper, "-$&").replace(/^-/, "").toLowerCase();
+  return string
+    .replace(upper, '-$&')
+    .replace(/^-/, '')
+    .toLowerCase();
 }
 
 module.exports = {
-    title: 'Synchro Charts',
-    theme: {
-        fontFamily: {
-            base: '"Amazon Ember", Helvetica, Arial, sans-serif'
-        }
+  title: 'Synchro Charts',
+  theme: {
+    fontFamily: {
+      base: '"Amazon Ember", Helvetica, Arial, sans-serif',
     },
-    styleguideDir: 'app',
-    pagePerSection: true,
-    sections: [
+  },
+  assetsDir: 'assets',
```

Comment:
This is the only change - `assetsDir` is a config property provided by `styleguidist` that directly copies the containing folder to the root build directory. I don't love the naming but we can consider CNAME to be a GitHub "asset". The alternative would be to modify the internal webpack configuration or write custom scripts.